lib/OpenQA/LiveHandler.pm

```diff
@@ -47,6 +47,7 @@ sub startup {
     my $self = shift;
 
     $self->defaults(appname => 'openQA Live Handler');
+    return if grep { /--help/ } @ARGV;
```

Comment:
Is this sensible to do within [startup](https://mojolicious.org/perldoc/Mojolicious#startup)? The docs don't really say anything about that. What about exiting with an error code?

Reply:
I'm not completely happy with this change, too. For example help can also be called with `-h`.

Reply:
If I would have found a better approach I would have done it :) I am happy to change if you have good suggestions. I could of course change the regex to `/^(-h|--help)$/`

Reply:
I suspect @kraih would be best to comment on that then?

Reply:
Certainly. But I am hesitant to ask individuals for review to not annoy :)

Reply:
No, this is not a good idea at all. Same reason you shouldn't add defaults to the app scripts.

Reply:
And there are more ways to do `--help`, like `./script/foo help daemon` and the like.